pkgs/by-name/ea/eaphammer/package.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  python3Packages,
+  makeWrapper,
+  apacheHttpd,
+  curl,
+  dnsmasq,
+  hcxdumptool,
+  hcxtools,
+  libnfnetlink,
+  libnl,
+  libpcap,
+  openssl,
+  responder,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "eaphammer";
+  version = "1.14.1";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "s0lst1c3";
+    repo = "eaphammer";
+    rev = "v${version}";
+    hash = "sha256-LUqUSfM4U5adOk5yaM05wTE/47iKXiA46FzSP3peivw=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildInputs = [
+    apacheHttpd
+    curl
+    dnsmasq
+    hcxdumptool
+    hcxtools
+    libnfnetlink
+    libnl
+    libpcap
+    openssl
+    responder
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    beautifulsoup4
+    flask
+    flask-cors
+    flask-socketio
+    gevent
+    lxml
+    pem
+    pyopenssl
+    pywebcopy
+    requests
+    scapy
+    tqdm
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,share/eaphammer}
+    rm README.md *.txt LICENSE
+    cp -a * $out/share/eaphammer/
+    makeWrapper ${python3}/bin/python $out/bin/eaphammer \
+      --add-flags "$out/share/eaphammer/eaphammer" \
+      --prefix PYTHONPATH : ${python3Packages.makePythonPath propagatedBuildInputs}
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Targeted evil twin attacks against WPA2-Enterprise networks. Indirect wireless pivots using hostile portal attacks";
```

Comment:
description should be only one sentence https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

Reply:
Fixed